README.md

```diff
@@ -2,21 +2,48 @@
 
 ## Overview
 
-These are connectors that have been developed for Workspace One Mobile Flows. Although all of them can be used as-is, 
+These are connectors that have been developed for Workspace One Mobile Flows. Although all of them can be used as-is,
 they can also be used as base from which VMware customers can develop their own connectors.
 
-The connectors are written in Java and use the [Spring Framework](https://spring.io/). More specifically, they use 
+The connectors are written in Java and use the [Spring Framework](https://spring.io/). More specifically, they use
 [Spring Boot](https://projects.spring.io/spring-boot/), embedding [Tomcat](http://tomcat.apache.org/) 8.5.
 
-For a detailed, language-neutral, specification for how to develop connectors, please see the 
+For a detailed, language-neutral, specification for how to develop connectors, please see the
 [Card Connectors Guide](https://github.com/vmwaresamples/card-connectors-guide).
 
 This repository also includes common libraries. Please see their [README](https://github.com/vmware/connectors-workspace-one/blob/master/common/README.md) for more details.
 
+## Docker
+
+The connectors can be run as [Docker](https://www.docker.com/) containers.
+
 ### Prerequisites
 
-* Java 8
-* Vagrant (use [Vagrantfile](https://github.com/vmware/connectors-workspace-one/blob/master/Vagrantfile))
+* Docker
+
+### Running
+
+The latest published versions of the connectors are available in the [Docker Hub registry](https://hub.docker.com/u/ws1connectors/).
+
+For example, you can run the latest jira-connector with:
+
+```
+docker run --name jira-connector \
+           -p 8080:8080 \
```

Comment:
TODO: add an example of getting the container app's port wrong on the docker command to the troubleshooting

Reply:
done